devops/README.md

```diff
@@ -0,0 +1,71 @@
+`docker build devops/docker/ubuntu20.04-amd64 -t osconfig-ubuntu2004-amd64`
+
+This builds a Ubuntu 20.04 amd64 variant and tags the image with _osconfig-ubuntu2004-amd64_
+
+# Build Systems and Pipelines
+OSConfig makes use of [Azure Pipelines](https://azure.microsoft.com/services/devops/pipelines/) for many tasks inclufing building OSConfig's binaries, unit tests, end-to-end tests amongst others. All of OSConfig's pipelines are contained under `devops/pipelines`.
+
+ * [binary_package.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/binary_package.yaml) - Produces Debian and source packages and also runs unit-tests
+ * [cleanup_e2etest_infra.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/cleanup_e2etest_infra.yaml) - Cleans up end-to-end test infrastructure
+ * [create_containers.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/create_containers.yaml) - Creates the container environments and pushes the container to your image repository
+ * [e2etest.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/e2etest.yaml) - Performs end-to-end testing of OSConfig (Setting up IoTHub, DPS, OSConfig Hosts)
+ * [source_packaging.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/source_packaging.yaml) - Creates a source package and runs unit-tests
+
+## Creating your own pipeline for packaging/testing
+In order to setup pipelines, the following is prerequisites are necessary.
+
+### Prerequisites
+ * [Azure Pipelines](https://azure.microsoft.com/services/devops/pipelines/) instance
+ * [Azure Container Repository (ACR)](https://azure.microsoft.com/services/container-registry/) - or any other container repository
+
+### Creating the Service Connection
+The pipelines need to authenticate against the container repository, this is done by [creating a service connection](https://docs.microsoft.com/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#create-a-service-connection) to a [Docker Registry or others](https://docs.microsoft.com/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#docker-hub-or-others).
+
+### Creating the Pipelines
+In order for packaging/testing to be automated, only two pipelines are required to be onboarded (See [Create your first pipeline](https://docs.microsoft.com/azure/devops/pipelines/create-first-pipeline?view=azure-devops) for more info):
+
+ * [create_containers.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/create_containers.yaml)
+ * [binary_package.yaml](https://github.com/Azure/azure-osconfig/blob/main/devops/pipeline/binary_package.yaml)
+
+ Both pipelines require the following variables in order to work against your service connection and container repository. Make sure the `SERVICE_CONNECTION` points to the repository (case-sensitive) service connection created above and the `CONTAINER_REGISTRY` URI for the image repository.
```

Comment:
Here and elsewhere, it may be better to avoid saying *"you"* and maintain a declarative voice.

Reply:
Essentially to state *what/how* rather and avoid pronouns like you/your. For example:

*...in order to work against **the/a** service connection...*